src/main/java/com/spotify/annoy/ANNIndex.java

```diff
@@ -8,53 +8,69 @@
 import java.util.*;
 
 /**
- * Read-only Approximate Nearest Neighbor Index which queries databases created by annoy.
+ * Read-only Approximate Nearest Neighbor Index which queries
+ * databases created by annoy.
  */
-public class ANNIndex implements AnnoyIndex {
```

Comment:
Even with Guice, we need this interface to inject a mock at test time... @nicolamontecchio can you add the interface back so that we can merge your branch?